contracts/strategies/wrapped-voting/solmate/ERC721.sol

```diff
@@ -0,0 +1,207 @@
+// SPDX-License-Identifier: AGPL-3.0-only
+pragma solidity >=0.8.0;
```

Comment:
are we changing anything on the solady contract ? 
if not, let's use it from the library directly  
if yes -> let's document the changes (like how we did for EAS resolver)